test/mi.c

```diff
@@ -629,6 +629,44 @@ static void test_admin_invalid_formats(nvme_mi_ep_t ep)
 	assert(rc != 0);
 }
 
+static void test_mi_invalid_formats(nvme_mi_ep_t ep)
+{
+	struct {
+		struct nvme_mi_mi_req_hdr hdr;
+		uint8_t data[4];
+	} req = { 0 };
+	struct nvme_mi_mi_resp_hdr resp = { 0 };
+	nvme_mi_ctrl_t ctrl;
+	size_t len;
+	int rc;
+
+	test_set_transport_callback(ep, test_admin_invalid_formats_cb, NULL);
```

Comment:
Might be good to rename the callback if we're no longer just using it for admin.

Maybe something like `test_rejected_command_cb` ?